lib/content-services/src/lib/search/components/search-header/search-header.component.html

```diff
@@ -5,7 +5,7 @@
             #menuTrigger="matMenuTrigger"
             (click)="onMenuButtonClick($event)"
             class="adf-filter-button"
-            matTooltip="{{ 'SEARCH.SEARCH_HEADER.FILTER_BY' | translate: { category: col.title || 'Type' } }}">
+            matTooltip="{{ 'SEARCH.SEARCH_HEADER.FILTER_BY' | translate: { category: (col.title | translate) || 'Type' } }}">
```

Comment:
you are doing double translation, and fallback value 'Type' is not translated. Better create a function that returns tooltip value.